frontend/javascripts/oxalis/shaders/main_data_shaders.glsl.ts

```diff
@@ -222,14 +226,33 @@ void main() {
         color_value = abs(color_value - <%= name %>_is_inverted);
         // Catch the case where max == min would causes a NaN value and use black as a fallback color.
         color_value = mix(color_value, vec3(0.0), is_max_and_min_equal);
-        // Multiply with color and alpha for <%= name %>
-        data_color += color_value * <%= name %>_alpha * <%= name %>_color;
+        color_value = color_value * <%= name %>_alpha * <%= name %>_color;
+        // Marking the color as invalid by setting alpha to 0.0 if the fallback color has been used
+        // so that it does not cover other colors.
+        vec4 layer_color = vec4(color_value, used_fallback == 1.0 ? 0.0 : maybe_filtered_color.a * <%= name %>_alpha);
```

Comment:
I think, I'd simply rename `color_value` to `layer_value` and then don't define a new variable (as it's not necessary to have both at the same time, I think).